doc/source/whatsnew/v0.18.0.txt

```diff
@@ -161,6 +161,30 @@ In addition, ``.round()`` will be available thru the ``.dt`` accessor of ``Serie
 
 .. _whatsnew_0180.api_breaking:
 
+
+Changed to_string(index=False) method of DataFrame and Series formatter class.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- the "leading whitespaces" have been removed from the output of "to_string(index=False)" method (:issue:'11833')
```

Comment:
on 2nd thought, let's just use this single line 168 instead, pls remove the section. also put double backticks around ``to_string(index=False)``